scripts/linux/compile.sh

```diff
@@ -38,13 +38,13 @@ fi
 if [ -d $parent_dir/output ]; then
 	read -p "Directory $parent_dir/output already exists. The script will delete and recreate it. Is that okay? (y/N) : " yn
 	case $yn in
-	    [Yy]* ) rm -rf $parent_dir/output ;;
+	    [Yy]* ) echo 1 ;;
```

Comment:
Should this remain in the PR? It goes against the logic of the script - user is asked if it's okay to delete the directory.

Reply:
Sorry should remain as the original